protocol/src/main/java/org/apache/drill/exec/proto/beans/QueryContextInformation.java

```diff
@@ -51,6 +51,7 @@ public static QueryContextInformation getDefaultInstance()
     private int timeZone;
     private String defaultSchemaName;
     private String sessionId;
+    private int hllAccuracy;
```

Comment:
hllAccuracy is required within the HLL UDFs. The only way I found to pass information inside the UDFs is by injecting the `@Inject ContextInformation` available in the `UDFUtilities.java`.
Hence, I define it in the QueryContext eventually passing it to the `ContextInformation`. Please suggest if there is a better way to do the same.hllAccuracy is required within the HLL UDFs. The only way I found to pass information inside the UDFs is by injecting the `@Inject ContextInformation` available in the `UDFUtilities.java`.
Hence, I define it in the QueryContext eventually passing it to the `ContextInformation`. Please suggest if there is a better way to do the same.